### PR TITLE
New page icons and update next to 14.2

### PR DESCRIPTION
**Pull request by amorimjj**

### CODE REVIEW

File: packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.en.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+slug: shortcuts
+group: shortcuts
+order: 5
+---
+
+## Em ação
+
+### Uso da variante dark
+
+Use a variante `dark:` para definir estilos no modo escuro.
```
Comment (programad) [inline packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.en.mdx:13]: Is this text correct for this page?

File: packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.pt-br.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+slug: atalhos
+group: shortcuts
+order: 5
+---
+
+## Em ação
+
+### Uso da variante dark
+
+Use a variante `dark:` para definir estilos no modo escuro.
```
Comment (programad) [inline packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.pt-br.mdx:13]: Is this text correct for this page?

File: packages/ignition/src/app/locales/docs/usage/shortcuts.en.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+import RocketIconsText, { RocketIconsTextDefault } from "@/components/rocketicons-text";
+
+## Usage
+
+# What is shortcuts?
+
+Using <RocketIconsText /> to style icons on dark mode.
+
+You can style icons differently when dark mode is enabled.
```
Comment (programad) [inline packages/ignition/src/app/locales/docs/usage/shortcuts.en.mdx:16]: Is this text correct for this page? The pt-br one seems to be OK.

File: packages/tailwind/src/theme.ts [hunk 4]
```diff
@@ -8,8 +8,8 @@ const theme: ThemeOptions = {
     outlined: ""
   },
   sizes: {
-    xs: "h-2 w-2",
-    sm: "h-4 w-4",
+    xs: "h-2",
```
Comment (programad) [inline packages/tailwind/src/theme.ts:11]: I don't get why these two lost the w while the others kept it.

**Review by programad — CHANGES_REQUESTED** [review]
A few observations and I think we need at least one change in docs text.

File: packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.en.mdx (hunk 1 above)
Comment (amorimjj) [reply packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.en.mdx]: I added that by mistake. Removed

File: packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.pt-br.mdx (hunk 2 above)
Comment (amorimjj) [reply packages/ignition/src/app/locales/docs/usage/components/shortcuts-action.pt-br.mdx]: I've added that by mistake. Removed

File: packages/ignition/src/app/locales/docs/usage/shortcuts.en.mdx (hunk 3 above)
Comment (amorimjj) [reply packages/ignition/src/app/locales/docs/usage/shortcuts.en.mdx]: Fixed

File: packages/tailwind/src/theme.ts (hunk 4 above)
Comment (amorimjj) [reply packages/tailwind/src/theme.ts:11]: Keeping both on small size was making icons disappearing on small screen. We should investigate that on future

**Review by programad — APPROVED** [review]
Code 👍

**Pull request merged**